frontend/src/AlertsManagement/AlertsList.js

```diff
@@ -47,7 +47,8 @@ export default function AlertsList(props) {
                 disableTypography
                 primary={<Typography variant="body1" >Alert on 
                   <Box fontWeight='fontWeightBold' display='inline' m={1} style={{ color: '#0FA3B1'}}>
-                    {` ${allAlerts.get(alertId).timestampDate} (${formatDate(allAlerts.get(alertId).timestampDate)})`}
+                    {` ${allAlerts.get(alertId).timestampDate} (
+                      ${formatDate(allAlerts.get(alertId).timestampDate, /** mFirst = */ true)})`}
```

Comment:
the name mFirst is a bit unclear on what it is by the way.

the trade-off of longer variable name (i assume without looking at any other code that 'm' = month in this case?